platform/android/os_android.cpp

```diff
@@ -748,6 +760,18 @@ void OS_Android::vibrate_handheld(int p_duration_ms) {
 	godot_java->vibrate(p_duration_ms);
 }
 
+String OS_Android::get_data_path() const {
+	return "/sdcard/godot";
```

Comment:
The path here is different from the path in the stacktrace you posted.
Are you running into the same error when storing a test project under this path?

Reply:
/storage/.../0/ is the same in android as saying /sdcard, it shouldn't matter, but I can test again without it (this is the defaualt project path for some reason, it didn't bother me so I didn't took the time to check why)

Reply:
BTW about that, I will probebly change it to get intenal memory, and let the /goot be added automaticly through getGodotDirectory because curently it saves the data to /sdcard/godot/godot.
and also I think to change root directory to also be internal memory, is it ok?